transformer_engine/pytorch/module.py

```diff
@@ -323,7 +335,12 @@ def fp8_init(self, num_gemms: int = 1) -> None:
         # Allocate scales and amaxes
         self.init_fp8_meta_tensors()
 
-    def pre_forward(self, inp: torch.Tensor, num_gemms: int = 1) -> None:
+    def pre_forward(
```

Comment:
Be advised that https://github.com/NVIDIA/TransformerEngine/pull/50 changes this function into a context manager: https://github.com/ptrendx/TransformerEngine/blob/e9d85b8437c0a1701703a4257c8201d2be83a931/transformer_engine/pytorch/module.py#L365